src/index.ts

```diff
@@ -279,35 +283,35 @@ const legacyTheme = (
     forceSingleTheme = undefined,
     forceEmptyThemeSelectors
   } = options;
-  let newRules: postcss.Rule[] = [];
+  let newRules: Rule[] = [];
 
-  root.walkRules(rule => {
-    const themedDeclarations: postcss.Declaration[] = [];
+  root.walkRules(walkRule => {
```

Comment:
<img width="894" alt="image" src="https://user-images.githubusercontent.com/5761061/97637204-72e29300-19f7-11eb-9f99-445c48ceb59e.png">

Looking at the [migration guide](https://evilmartians.com/chronicles/postcss-8-plugin-migration), I think we should remove the `root.walk` and use `Rules` api instead.